content/en/blog/2022/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-03-15
+slug: k8s-triage-bot-helper-ci-job
+---
+
+# K8s Triage bot Helper CI Job: ‘*make update*’
+
+## Introduction 
+
+If you are contributing to the Kubernetes project and are developing on a Windows PC, it is conceivable that you will encounter certain issues that will cause your patches to master to somehow be halted. This blog is a workaround for a similar issue I encountered when attempting to have my modifications approved and merged into the master repository.
+
+## Why is this needed?
+ 
+While contributing to [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) for some minor documentation changes, the pushed changes needed to be updated with other verified contents of the entire documentation. So, in order for the change to take effect, a single command must be performed to ensure that all tests on the CI pipeline pass. The single command `make update` runs all presubmission verification tests. For some reason on the windows `wsl` environment the tests, specifically the [update-openapi-spec.sh](https://github.com/SubhasmitaSw/kubernetes/blob/master/hack/update-openapi-spec.sh) script, failed (in my case, take a look at the conversation [here](https://github.com/kubernetes/kubernetes/pull/107691)), eventually failing the `pull-kubernetes-verify` tests.
+
+> Advertently, the test cases pass to generate the updated files in other beefy Operating Systems like mac or Linux.
```

Comment:
was this line supposed to be edited out?

Reply:
According to me, this provides a piece of background information on where this error possibly won't occur

Reply:
This isn't a block quote, block quote markup isn't appropriate.

Can you reword? I don't understand what you wanted to convey.

Reply:
I still don't understand what the line means, and it doesn't seem necessary for the article.

Reply:
I'll remove it.